core/tests/protractor_utils/workflow.js

```diff
@@ -313,20 +317,17 @@ var uploadImage = async function(
   await waitFor.visibilityOf(
     imageClickableElement,
     'Image element is taking too long to appear.');
-  await imageClickableElement.click();
+  await action.click('Image Clickable Element', imageClickableElement);
 
   if (resetExistingImage) {
     expect(await thumbnailResetButton.isPresent()).toBe(true);
-    await waitFor.elementToBeClickable(
-      thumbnailResetButton,
-      'Topic thumbnail reset button taking too long to appear.');
-    await thumbnailResetButton.click();
+    await action.click('Thumbnail Reset Button', thumbnailResetButton);
   } else {
     expect(await thumbnailResetButton.isPresent()).toBe(false);
   }
 
   absPath = path.resolve(__dirname, imgPath);
-  return await imageUploadInput.sendKeys(absPath);
+  return await action.sendKeys(imageUploadInput, absPath);
```

Comment:
You're missing the first argument to `sendKeys`: a string with the name of the element